tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py

```diff
@@ -258,7 +287,10 @@ def test_pfc_pause_single_lossless_prio_reboot(snappi_api,                  # no
                  test_traffic_pause=True,
                  snappi_extra_params=snappi_extra_params)
 
-    cleanup_config(dut_list, snappi_ports)
+    # Teardown config through a reload
+    logger.info("Reloading config to teardown")
+    for duthost in dut_list:
+        config_reload(sonic_host=duthost, config_source='config_db', safe_reload=True)
```

Comment:
fixed

Reply:
In addition, if the DUTs are simply same DUT, but different ASIC, this code is simply re-loading config 2 times in the same DUT. This can be optimized in this case.

Reply:
@rraghav-cisco yes this block has been removed